src/colvardeps.h

```diff
@@ -257,6 +257,8 @@ class colvardeps {
     f_cvb_scale_biasing_force,
     /// \brief whether this bias is applied to one or more ext-Lagrangian colvars
     f_cvb_extended,
+    /// Process this bias's data in parallel over multiple CPU threads
+    f_cvb_smp,
```

Comment:
@giacomofiorin Should we further divide the SMP case into flags like `f_cvb_smp_omp`, `f_cvb_smp_ckloop`, `f_cvb_smp_cuda` and `f_cvb_smp_sycl`? What about the Colvars components? Is it possible to also have `f_cvc_smp_*`, or reuse the `f_cvb_smp` for CVCs?